### ISSUE TITLE
Does this package support "inline reply actions" in Notifications

### ISSUE THREAD

**Issue by JudyHsiao** [evollu/react-native-fcm#395]
I would like to ask does this package support "inline reply actions" as in 
https://developer.android.com/guide/topics/ui/notifiers/notifications.html#direct

Thank you very much for your reply!

**Comment by evollu** [comment]
no. but you should be able to add this feature in FIRLocalMessagingHelper
